app/assets/javascripts/live-search.js

```diff
@@ -40,6 +40,7 @@
       }
     },
     pageTrack: function(){
+      ga("set", "page", window.location.pathname + window.location.search);
       GOVUK.analytics.setResultCountDimension(liveSearch.cache().result_count);
       GOVUK.analytics.trackPageview();
```

Comment:
That's what I wanted to do at first, but this only works for this specific pageview, but not for the event that records the positions of the search results, [which is called via the `liveSearch.pageTrack` event](https://github.com/alphagov/frontend/blob/112eb28c5550b42af000f34772d496ec09103564/app/assets/javascripts/search.js#L125).

`GOVUK.analytics.trackEvent` unfortunately doesn't support forwarding the `page` variable.